Source/Scene/InvertClassification.js

```diff
@@ -0,0 +1,369 @@
+define([
+        '../Core/Color',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/destroyObject',
+        '../Core/PixelFormat',
+        '../Renderer/ClearCommand',
+        '../Renderer/Framebuffer',
+        '../Renderer/PixelDatatype',
+        '../Renderer/RenderState',
+        '../Renderer/Sampler',
+        '../Renderer/ShaderSource',
+        '../Renderer/Texture',
+        '../Renderer/TextureMagnificationFilter',
+        '../Renderer/TextureMinificationFilter',
+        '../Renderer/TextureWrap',
+        '../Shaders/PostProcessFilters/PassThrough',
+        './BlendingState',
+        './StencilFunction',
+        './StencilOperation'
+    ], function(
+        Color,
+        defined,
+        defineProperties,
+        destroyObject,
+        PixelFormat,
+        ClearCommand,
+        Framebuffer,
+        PixelDatatype,
+        RenderState,
+        Sampler,
+        ShaderSource,
+        Texture,
+        TextureMagnificationFilter,
+        TextureMinificationFilter,
+        TextureWrap,
+        PassThrough,
+        BlendingState,
+        StencilFunction,
+        StencilOperation) {
+    'use strict';
+
+    /**
+     * @private
+     */
+    function InvertClassification() {
```

Comment:
Is it reasonable to have a unit test for this file?

If not, how is the current coverage?

Reply:
Coverage is at 91%. The only thing not covered is the destroy function.